test/lib/linter/rules/snapshots/renderer.ts.md

```diff
@@ -43,6 +43,14 @@ Generated by [AVA](https://avajs.dev).
         ],
         warningCount: 0,
       },
+      {
+        coverageInfo: [],
+        errorCount: 0,
+        fatalErrorCount: 0,
+        filePath: '12Control.js',
```

Comment:
Shouldn't this be reported?

Reply:
Oh, I've missed that. Thanks for noticing 👍🏻